src/components/ZclClusterView.vue

```diff
@@ -222,20 +226,10 @@ export default {
     setIndividualClusterFilterString(filterString) {
       this.$store.dispatch('zap/setIndividualClusterFilterString', filterString)
     },
-    onResize(size) {
```

Comment:
Why is this removed?

Reply:
it is not needed anymore. It was necessary for the table's height in popup.